app/src/main/java/com/duckduckgo/app/browser/duckplayer/DuckPlayerJSHelper.kt

```diff
@@ -127,6 +137,9 @@ class DuckPlayerJSHelper @Inject constructor(
                 jsonObject.put("platform", JSONObject("""{ name: "android" }"""))
                 jsonObject.put("locale", java.util.Locale.getDefault().language)
                 jsonObject.put("env", if (appBuildConfig.isDebug) "development" else "production")
+
+                // Custom Error Settings
+                jsonObject.getJSONObject("settings").put("customError", getCustomErrorSettings())
```

Comment:
We shouldn't pass this from native when CSS already has access to the RC JSON. See `DuckPlayerContentScopeConfigPlugin`

Reply:
Maybe I'm misunderstanding how `DuckPlayerContentScopeConfigPlugin` works, but in the case of CustomError, the setting needs to be passed into the special page through the initialSetup call for consistency (since the special page doesn't have access to RC on other platforms). Is there a middle ground where we can still pass this to initialSetup rather than have C-S-S access it directly?

Reply:
@CrisBarreiro  I've finished making the code changes from the call -- please let me know if I left anything out. Thank you!